src/components/SaveExport/ExportDialog.jsx

```diff
@@ -164,6 +241,43 @@ export default () => {
                     <Col sm={8}>
                         <Card className="h-100">
                             <Card.Body>
+                                <h2>Area to export</h2>
```

Comment:
The `ExportDialog.jsx` file is already quite big, maybe this could be moved into it's own component?